go.mod

```diff
@@ -64,10 +66,10 @@ require (
 	github.com/go-openapi/validate v0.19.2
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
 	github.com/godbus/dbus v4.1.0+incompatible
-	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
+	github.com/gogo/protobuf v1.2.0
```

Comment:
in particular this seems like a big change, and will also require more than a simple `hack/update-vendor.sh`

Reply:
https://github.com/kubernetes/kubernetes/pull/79261 is in flight and updates this to v1.1.1

I don't have the expertise to know if there are changes beyond the generated .pb.go files that we want or don't want